static/app/views/insights/common/components/modulePageFilterBar.tsx

```diff
@@ -59,8 +59,11 @@ export function ModulePageFilterBar({moduleName, onProjectChange, extraFilters}:
           position="bottom-start"
           disabled={!showTooltip}
         >
-          <ProjectPageFilter onChange={onProjectChange} />
+          {/* TODO: Placing a DIV here is a hack, it allows the tooltip to close and the ProjectPageFilter to close at the same time,
+          otherwise two clicks are required because of some rerendering/event propogation issues into the children */}
+          <div style={{width: '100px', position: 'absolute', height: '100%'}} />
```

Comment:
🤔 I tested this out locally, and the fix still seems to work if I just removed the invisible div element, ie no children in the tooltip. Maybe check if it also works for you locally as well?

Reply:
Ohh interesting, iirc the tooltip didn't point to the right element (the project dropdown selector) when i tried this, but i could be wrong.
Did you notice that?

Reply:
I double checked, and the tooltip is pointing in a slightly different spots.
Without div
![image](https://github.com/user-attachments/assets/1500a0d1-c94c-45c0-a3f5-b5b4d78f4813)

With div
![image](https://github.com/user-attachments/assets/964f8d76-a43a-452d-9010-c8212cc5f03d)

With that in mind, i'll just keep the div for now just to add the mocks a bit better.

Reply:
Ah good point, thanks for checking!